doc/conf.py

```diff
@@ -66,13 +66,14 @@
 }
 
 autodoc_mock_imports = [
-    'iris',
-    'stratify',
-    'ESMF',
     'cartopy',
     'cf_units',
+    'ESMF',
+    'geopy',
+    'iris',
+    'nested_lookup',
```

Comment:
quick question - I see this in `main` too so no problem for env building and such, but I don't remember it before, and both @zklaus and myself are preaching we get as many packages from conda-forge rather than PyPi, this one is on c-f https://anaconda.org/conda-forge/nested-lookup so how about we move it in `environment.yml`? In a separate PR, that is. Any reason why it must be off PyPi?

Reply:
Is this really related to PyPi vs. conda? I thought these lines just make sure that imports are mocked during the build of the autodoc.

Reply:
yeah! no, but that package is in `setup.py`, didn't bother commenting there

Reply:
Ah, got you.